Peach/peach.py

```diff
@@ -0,0 +1,46 @@
+import requests
+import datetime
+
+class Peach(object):
+    def __init__(self,key):
+        self.key = key
+
+
+    def fetch(self,url):
+        r = requests.get(url)
+        data = r.json()
+        return data
+
+    def getName(self,name):
+        uuid = self.fetch(url=f'https://api.mojang.com/users/profiles/minecraft/{name}')['id']
+        hyapi = self.fetch(f'https://api.hypixel.net/player?key={self.key}&uuid={uuid}')
```

Comment:
don't use key in param, use "Api-key" header, see docs @ https://api.hypixel.net

Reply:
why though?

Reply:
> don't use key in param, use "Api-key" header, see docs @ https://api.hypixel.net

Why would we use "Api-key" when "key" is perfectly fine? I used this myself in my Project "HyDrp" and it works

Reply:
Headers help manage rate limiting. See the "Limits" section @ https://api.hypixel.net/

Reply:
Using param works fine, but it's not documented on the api docs anymore, so it's better to just follow what the docs say.